internal/daysteps/daysteps.go

```diff
@@ -13,8 +20,41 @@ const (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// TODO: реализовать функцию
+	parts := strings.Split(data, ",")
+
+	if len(parts) == 2 {
+		steps, err := strconv.Atoi(parts[0])
+		if err != nil {
+			return 0, 0, err
+		}
+		if steps <= 0 {
+			return 0, 0, errors.New("cannot be equal to 0")
+		}
+		duration, err := time.ParseDuration(parts[1])
+		if err != nil {
+			return 0, 0, err
```

Comment:
Лучше оборачивать ошибки
```suggestion
			return 0, 0, fmt.Errorf("invalid duration format: %w", err)
```